### ISSUE TITLE
Action Pad and Action - Provide a valid label for form fields - (2036569766)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5558]
### Module: 
 06 Action Pad and Action 

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036569766 

### Severity: 
 10 

### Description: 
 There are buttons with identical accessible names. Examples include:

- Undo action, for display purposes only
- Redo action, for display purposes only
- Expand

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Undo action, for display purposes only" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  <div class="text-container">Undo</div>
</button>	
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with internal text, an aria-label attribute, or an aria-labelledby attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Undo action pad, for display purposes only" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  <div class="text-container">Undo</div>
</button>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Similar to #5555. The issue is a result of the component tests for the audit, not the component itself.

No further action is needed.
